docs/tutorials/deploy-vm/cloudinit.md

```diff
@@ -1,130 +1,20 @@
 # Deploy a VM With Cloud-Init
 
-// TODO ([github.com/vmware-tanzu/vm-operator#122](https://github.com/vmware-tanzu/vm-operator/issues/122))
+This page reviews deploying a VM with the bootstrap method Cloud-Init.
 
-This page reviews the different components, workflows, and decisions related to deploying a VM with VM Operator:
+## Description
 
-## The VirtualMachine API
+[Cloud-Init](https://cloudinit.readthedocs.io/en/latest/) is widely recognized as the de facto method for bootstrapping modern VM instances on hyperscalers, including Tanzu with VM Operator.
```

Comment:
```suggestion
[Cloud-Init](https://cloudinit.readthedocs.io/en/latest/) is widely recognized as the de-facto method for bootstrapping modern VM instances on hyperscalers, including VM Service.
```

Reply:
making it "including VM Service on vSphere with Tanzu" since we're talking about hyperscalers.